main.js

```diff
@@ -0,0 +1,51 @@
+// Math helpers
+function randomIntFromInterval(min, max) { // min and max included 
+    return Math.floor(Math.random() * (max - min + 1) + min)
+  }
+
+// Start button logic
+document.getElementById("startButton").addEventListener("click", function(){
+    let circlesElements = []
+    let bounceDiv = document.getElementById("bounceDiv");
+
+    let xCoord = Math.random()*100
+
+    let {top:topBounceDiv,left:leftBounceDiv,bottom:bottomBounceDiv,right:rightBounceDiv} = bounceDiv.getBoundingClientRect();
+
+    let diffRightLeft = rightBounceDiv-leftBounceDiv
+    let difTopBottom = bottomBounceDiv - topBounceDiv
+
+    const newDiv = document.createElement("div");
+    const circleRadius = 40
+    newDiv.style.width=circleRadius+"px"
+    newDiv.style.height=circleRadius+"px"
+    newDiv.style.backgroundColor="red"
+    newDiv.style.borderRadius="100%"
+    newDiv.style.position="absolute"
```

Comment:
I would extract the styles that are not 'configurable' into a css class and assign it to newDiv